tests/e2e/default/reports/bulk-delete.wdio-spec.js

```diff
@@ -2,58 +2,13 @@ const utils = require('@utils');
 const commonElements = require('@page-objects/default/common/common.wdio.page');
 const reportsPage = require('@page-objects/default/reports/reports.wdio.page');
 const loginPage = require('@page-objects/default/login/login.wdio.page');
+const reportFactory = require('@factories/cht/reports/generic-report');
 
 describe('Bulk delete reports', () => {
   const docs = [
-    {
-      fields: {
-        lmp_date: 'Feb 3, 2016'
-      },
-      form: 'P',
-      type: 'data_record',
-      content_type: 'xml',
-      reported_date: 1462333250374,
-      contact: {
-        _id: '3305E3D0-2970-7B0E-AB97-C3239CD22D32'
-      },
-      from: '+555',
-      hidden_fields: []
-    },
-    {
-      fields: {
-        lmp_date: 'Feb 15, 2016'
-      },
-      form: 'P',
-      type: 'data_record',
-      content_type: 'xml',
-      reported_date: 1462338250374,
-      contact: {
-        _id: '3305E3D0-2970-7B0E-AB97-C3239CD22D32'
-      },
-      from: '+555',
-      hidden_fields: []
-    },
-    {
-      fields: {
-        ok: 'Yes!'
-      },
-      form: 'V',
-      type: 'data_record',
-      content_type: 'xml',
-      reported_date: 1462538250374,
-      contact: {
-        _id: '3305E3D0-2970-7B0E-AB97-C3239CD22D32'
-      },
-      from: '+555',
-      hidden_fields: []
-    },
-    {
-      name: 'Sharon',
```

Comment:
I removed it because the test is only selecting reports to delete them, and I didn't think it was important. But maybe there is something that I am not seeing, I can definitely add one person with that phone number just to be sure.
Thanks Josh!